File: Paladins.Api/Paladins.Api/Paladins.Common/Models/ItemModel.cs
using Paladins.Common.Interfaces.Definitions;
using System;

namespace Paladins.Common.Models
{
    public class ItemModel : IModel
    {
        public int Id { get; set; }
        public int PaladinsItemId { get; set; }
        public string Description { get; set; }
        public string ShortDescription { get; set; }
        public string DeviceName { get; set; }
        public int PaladinsChampionId { get; set; }
        public Uri ItemIconUrl { get; set; }
        public int Price { get; set; }
        public string ItemType { get; set; }
    }
}

File: Mobile/app/src/main/java/com/fivegearszerochill/paladins/domain/models/initial/ItemModel.kt
package com.fivegearszerochill.paladins.domain.models.initial

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ItemModel(
    @Expose
    @SerializedName("id")
    val id: Long,
    @Expose
    @SerializedName("paladinsItemId")
    val paladinsItemID: Long,
    @Expose
    @SerializedName("description")
    val description: String,
    @Expose
    @SerializedName("shortDescription")
    val shortDescription: String,
    @Expose
    @SerializedName("deviceName")
    val deviceName: String,
    @Expose
    @SerializedName("paladinsChampionId")
    val paladinsChampionID: Long,
    @Expose
    @SerializedName("itemIconUrl")
    val itemIconURL: String,
    @Expose
    @SerializedName("price")
    val price: Long,
    @Expose
    @SerializedName("itemType")
    val itemType: String
)